profiler/shakesapp/shakesapp/shakesapp.proto

```diff
@@ -0,0 +1,36 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// To update shakesapp.pb.go, run
+// protoc --go_out=plugins=grpc:. shakesapp.proto
+// --plugin=/path/to/protoc-gen-go
```

Comment:
Please make it so the command can be copy-pasted. You can either escape the newline, but the easiest is make it one line.

Reply:
Done.